forte2/ci/ci.py

```diff
@@ -163,6 +163,8 @@ def run(self):
         self.ci_sigma_builder.set_memory(self.ci_builder_memory)
         self.ci_sigma_builder.set_algorithm(self.ci_algorithm)
 
+        logger.log(f"Using CI algorithm: {self.ci_sigma_builder.get_algorithm()}", self.log_level)
```

Comment:
this was my omission, but can we add 1) possible values of `ci_algorithm` to the docstring and 2) sanity checking - `assert ci_algorithm.lower() in ["hz", "kh"]` (so that it can fail before `.run()` is called?